### ISSUE TITLE
librobot?

### ISSUE THREAD

**Issue by CptnGreen** [CptnGreen/robotics21#11]
В тот момент, когда в распоряжении участника бассейна окажется подвижный программируемый робот, ему нужно начать собирать собственную библиотеку функций для взаимодействия с ним.

Стоит ли пихать всё в одну библиотеку (скажем, `librobot`), или нужно выделить отдельные модули (`libultrasonic`, `libreflection`, ...)?

**Comment by bioidiad** [comment]
@CptnGreen Думаю можно пихать все в одну, просто разбить по модулям (отдельный хедер и сурсы для каждого компонента, который подключается к основному хедеру)

**Comment by CptnGreen** [comment]
@Freimor, что ты думаешь?

**Comment by Freimor** [comment]
Да, в одну библиотечку. Она врятли будет использоваться в дальнейшем

**Comment by CptnGreen** [comment]
@Freimor, в дальнейшем - это ты имеешь в виду в других проектах робо-ветки?

В любом случае, я бы не был так уверен, что не будет использоваться то же взаимодействие с датчиками.

**Comment by bioidiad** [comment]
@Freimor Я считаю, что бессмысленно делать библиотеку чтоб ее потом не использовать. Думается после бассейна должна быть та же политика что и в других ветках - минимум сторонних библиотек. Только так ты будешь полностью понимать что происходит)

**Comment by Freimor** [comment]
@bioidiad я напомню что после бассейна робототехника будет построена на ROS с его же окружением. В IoT из библиотечных функций может и можно будет использовать сенсоры и то сомнительно

**Comment by CptnGreen** [comment]
@Freimor, да, об этом я что-то не подумал

Ну тогда можно прототипы функций нашей библиотеки просто делать максимально похожими на прототипы функций из библиотек ROS чтобы облегчить дальнейший переход, да?
